getting-started/forge-multi-node.md

```diff
@@ -137,13 +137,18 @@ connections. Copy this hostname to use in the `add_data_node` command below.
 <img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/forge_images/timescale-forge-internal-dns.png" alt="Timescale Forge multi-node connection information"/>
 
 Once you have the **password** and **internal, Multi-node** hostname for each data node
-Service, connect to the access node using the `tsdbadmin` user and add a data node to
-your cluster.
+Service, connect to the access node using the `tsdbadmin` user.
 
-```SQL
+```bash
 psql -h {AN hostname} -p {port} tsdb tsdbadmin
 
-SELECT add_data_node('dn1', host => 'your_DN1_hostname', port => 5432 , password => 'tsdbadmin_user_password_for_DN1', bootstrap => false);
+```
+
+Then add a data node as follows.
+
+```sql
+SELECT add_data_node('dn1', host => 'your_DN1_hostname',
```

Comment:
FWIW, @niksajakovljevic brought up the point that eliding `bootstrap=>false` prints some notices about database and extension not being created. If we prefer to not print those notices, we can keep `bootstrap=>false`.

Generally, though, the intention is that you normally shouldn't have to care about bootstrapping. Disabling bootstrapping is only necessary for special cases where the default settings do not work.